benchmark/bm_entangling_layers.py

```diff
@@ -0,0 +1,43 @@
+"""Entangling layers benchmark.
+Creates an immutable QNode using the StronglyEntanglingLayers template,
+then evaluates it and its Jacobian.
+"""
+# pylint: disable=invalid-name
+import numpy as np
+
+import pennylane as qml
+from pennylane.templates.layers import StronglyEntanglingLayers
+from pennylane.templates.embeddings import AngleEmbedding
+from pennylane.init import strong_ent_layers_uniform
+
+n = 3
+n_min = 1
+n_max = 5
+n_step = 1
+
+n_wires = 3
+
+dev = qml.device("default.qubit", wires=n_wires)
+
+
+def circuit(weights, features=None):
+    """Quantum circuit."""
+    AngleEmbedding(features, range(n_wires))
+    StronglyEntanglingLayers(weights, wires=range(n_wires))
+    return qml.expval(qml.PauliZ(0))
+
+
+def benchmark(n_layers=3):
+    """Entangling layers"""
+
+    # print("circuit: {} layers, {} wires".format(n_layers, n_wires))
```

Comment:
Maybe have a "verbose" mode?

Reply:
Maybe. I just have this vague feeling that I've seen recommendations not to print stuff in benchmarks since printing might have an unpredictable latency, but I'm not sure.

Reply:
This is very anecdotal, but I have significantly improved IPython `%timeit` times in the past by removing printing

Reply:
Good points. Perhaps the default is to run the benchmarks with no printing (to get most realistic speed estimates), but still allow printing/logging somehow if the user desires? I imagine checking `if verbose` takes pretty minimal amount of time, so the check itself shouldn't unduly influence benchmark times.

On the other hand, maybe there's no need for printing at all? (I just mentioned it because I saw a lot of commented-out `print` statements)

Reply:
Added the verbose mode.